topsort-endpoints.yml

```diff
@@ -118,14 +147,31 @@ components:
         vendors:
           type: array
           description: An array of objects, each describing a vendor that should participate in the auction.
-          minItems: 0
+          minItems: 1
           items:
             $ref: '#/components/schemas/Vendor'
         session:
           $ref: '#/components/schemas/Session'
         bannerOptions:
           $ref: '#/components/schemas/BannerOptions'
 
+    AuctionRequestHomepageStandaloneBanners:
```

Comment:
I think we should add some more documentation to each of the request types and check how readme shows the info.

Have you seen how the generated code looks like?

Reply:
We've only seen how Swagger shows this. We don't really expect much from Readme; we want to render OpenAPI elsewhere and link from Readme.

Reply:
> I think we should add some more documentation to each of the request types and check how readme shows the info.
Yes. @roalonso is taking care of that. 

> Have you seen how the generated code looks like?
Yes. Swagger shows them perfectly fine.

Reply:
Actually when generating the code, this is what I get for the standalone banners:

```ts
export interface AuctionRequestHomepageStandaloneBanners {
  /**
   *
   * @type {Slots}
   * @memberof AuctionRequestHomepageStandaloneBanners
   */
  slots: Slots;
  /**
   *
   * @type {Session}
   * @memberof AuctionRequestHomepageStandaloneBanners
   */
  session: Session;
  /**
   *
   * @type {object}
   * @memberof AuctionRequestHomepageStandaloneBanners
   */
  bannerOptions: object;
}
```

which makes the acution request to be:

```ts
export interface AuctionRequest {
  /**
   *
   * @type {Slots}
   * @memberof AuctionRequest
   */
  slots: Slots;
  /**
   * An array of objects, each describing a product that should participate in the auction.
   * @type {Array<Product>}
   * @memberof AuctionRequest
   */
  products: Array<Product>;
  /**
   *
   * @type {Session}
   * @memberof AuctionRequest
   */
  session: Session;
  /**
   * An array of objects, each describing a vendor that should participate in the auction.
   * @type {Array<Vendor>}
   * @memberof AuctionRequest
   */
  vendors: Array<Vendor>;
  /**
   *
   * @type {object}
   * @memberof AuctionRequest
   */
  bannerOptions: object;
}
```

note how the banner options are not typed. and that you don't get a union.